src/mathutil.ts

```diff
@@ -0,0 +1,22 @@
+import { vec2 } from 'gl-matrix'
+
+export const lerp = (min: number, max: number, alpha: number): number => {
```

Comment:
Is `alpha` always gonna be `Math.random()`? I guess this a pretty generic form of `lerp`, but I wonder if it'd be nice to have one that always gave a random value in the specified range.

Reply:
I think lerp is quite a bit more generic than that. For example, we lerp the color of the wall based on the damage dealt to the wall.

Assuming that we will always need to have a non-random, basic `lerp()`, I think this might be one case where explicitness might be okay if it saves us an extra function. I don't see a big difference between the following:

```
lerp_random(min, max);
lerp(min, max, Math.random());
```